msteams-platform/toolkit/debug background process.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Debug background processes
+author: zyxiaoyuer
+description: Function of Visual studio code and Teams Toolkit during local debug
+ms.author: surbhigupta
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 03/03/2022
+---
+
+# Debug background process
+
+The local debug workflow involves the `.vscode/launch.json` and `.vscode/tasks.json` files to configure the debugger in Visual Studio Code, then Visual Studio Code launches the debuggers, and  Microsoft Edge or Chrome debugger launches a new browser instance as follows:
+
+1. The `launch.json` file configures the debugger in Visual Studio Code. Visual Studio Code runs the compound configurations `preLaunchTask`, defined as `Pre Debug Check`, and `Start All` in `.vscode/tasks.json` file.
+
+2. Visual Studio Code launches the debuggers specified in the compound configurations, such as **Attach to Bot**, **Attach to Backend**, **Attach to Frontend**, and **Launch Bot**.
+
+3. Microsoft Edge or Chrome debugger launches a new browser instance and opens a web page to load Teams client.
+
+## Prerequisites
+
+The following table lists the limitations if the required software is unavailable for debugging:
+
+|Project type|Installation| Limitation|
+|----------|--------------------------------|-----|
+|Tab without Azure functions | Node.js LTS versions 10, 12, **14 (recommended)**, 16 | The local debug terminates, if Node.js is not installed or the version doesn't match the requirement.|
+|Tab with Azure functions | Node.js LTS versions 10, 12, **14 (recommended)** |The local debug terminates, if Node.js is not installed or the version doesn't match the requirement.|
+|Bot | Node.js LTS versions 10, 12, **14 (recommended)**, 16|The local debug terminates, if Node.js is not installed or the version doesn't match the requirement.|
+|Messaging extension | Node.js LTS versions 10, 12, **14 (recommended)**, 16 |The local debug terminates, if Node.js is not installed or the version doesn't match the requirement.|
+|Sign in to Microsoft 365 account | Microsoft 365 credentials |Teams toolkit prompts you to sign in to Microsoft 365 account, if you haven't signed in. |
```

Comment:
This table is confusing because "Sign in to Microsoft 365 account", "NPM packages" are not "project type", right?
I suggest listing all prerequisites and their applicable scenario, like:
1. Node.js, applicable for tab, bot, messaging extension and Azure functions. Then list the version requirements for different project types. Other descriptions, etc.
2. Ngrok, applicable for bot and messaging extension. Other descriptions, etc.
3. etc.